charts/cert-manager-issuers/Chart.yaml

```
@@ -2,9 +2,9 @@ apiVersion: v2
 name: cert-manager-issuers
 description: Configure cert-manager Issuers and ClusterIssuers via Helm
 type: application
-version: 0.2.5
+version: 0.2.6
```

```suggestion
version: 0.3.0
```
Hi @sule26 

Thanks for the PR!

As it is a feat, I'd rather have a minor version bump pls. Thanks.

---

Done!